packaging/aznfs/DEBIAN/postinst

```diff
@@ -46,6 +46,11 @@ EOF
 
     sed -i '/AUTO_UPDATE_AZNFS/d' "$CONFIG_FILE"
 
+    if [ "$DEBIAN_FRONTEND" == "noninteractive" ]; then
+        echo "AUTO_UPDATE_AZNFS=true" >> "$CONFIG_FILE"
+        return
+    fi
+
```

Comment:
nit, move this block at L35 (before the title declaration), as that's more logical